core/controllers/async_download.py

```diff
@@ -167,36 +172,5 @@ def send_email_for_find_download(
             personalisation={
                 "download_url": download_url,
                 "find_service_url": find_service_url,
-                "file_format": file_format,
-                "file_size": file_size_str,
             },
         )
-
-
-def get_file_format_from_extension(file_extension: str) -> str:
-    """Return nice file format name based on the file extension.
-    :param file_extension: file extension,
-    :return: nice file format name,
-    """
-
-    file_format = ""
-    if file_extension == "xlsx":
-        file_format = "Microsoft Excel spreadsheet"
-    elif file_extension == "json":
-        file_format = "JSON file"
-    return file_format
-
-
-def get_human_readable_file_size(file_size_bytes: int) -> str:
-    """Return a human-readable file size string.
-    :param file_size_bytes: file size in bytes,
-    :return: human-readable file size,
-    """
-
-    file_size_kb = round(file_size_bytes / 1024, 1)
-    if file_size_kb < 1024:
-        return f"{round(file_size_kb, 1)} KB"
-    elif file_size_kb < 1024 * 1024:
-        return f"{round(file_size_kb / 1024, 1)} MB"
-    else:
-        return f"{round(file_size_kb / (1024 * 1024), 1)} GB"
```

Comment:
Have you moved these to the data frontend, or are we just removing them altogether?

Reply:
I'd be tempted to leave them here because some of your other tests are now broken. We can call them in the metadata endpoint to return human-readable values instead of raw content_length and content_type values?